### PR TITLE
Explain what a static variable does.

### PR DESCRIPTION
**Pull request by pavanagrawal123**
The current tutorial just adds in a static variable without explaining what it really does...

### CODE REVIEW

File: docs/csharp/quick-starts/introduction-to-classes.md [hunk 1]
```diff
@@ -100,7 +100,7 @@ Add the following member declaration to the `BankAccount` class:
 private static int accountNumberSeed = 1234567890;
 ```
 
-This is a data member. It's `private`, which means it can only be accessed by code inside the `BankAccount` class. It's a way of separating the public responsibilities (like having an account number) from the private implementation (how account numbers are generated.) Add the following two lines to the constructor to assign the account number:
+This is a data member. It's `private`, which means it can only be accessed by code inside the `BankAccount` class. It's a way of separating the public responsibilities (like having an account number) from the private implementation (how account numbers are generated.) It is also `static`, which means it is shared by all of the BankAccount objects. Non `static` variables are unique to each object. Add the following two lines to the constructor to assign the account number:
```
Comment (rpetrusha) [inline docs/csharp/quick-starts/introduction-to-classes.md]: Non \`static\` variables are --> The value of a non-static variable is

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for contributing to the dotnet/docs repo, @pavanagrawal123, and for defining `static`. I thought that your definition could be improved by making it clear that there is a separate value of a non-static variable for each instance.

**Comment by pavanagrawal123** [comment]
I've updated the change, let me know what you think.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Your changes look good, @pavanagrawal123. I'll merge your PR now. The updated article will be live on docs.microsoft.com in the next day or two.

**Pull request merged**